*** pt_BR.orig.aff	2015-12-06 19:21:21.526154225 -0200
--- pt_BR.aff	2015-12-06 19:28:49.546159523 -0200
***************
*** 1,3 ****
  SET ISO8859-1
! TRY áàãâéêíóõôúüçesianrtolcdugmphbyfvkwjqxz
  
--- 1,3 ----
  SET ISO8859-1
! 
  
***************
*** 15,16 ****
--- 15,32 ----
  
+ NAME    Brazilian Portuguese
+ VERSION 2.1.2
+ HOME	http://extensions.libreoffice.org/extension-center/vero-verificador-ortografico-e-hifenizador-em-portugues-do-brasil
+ AUTHOR  Raimundo Santos Moura
+ EMAIL   raimundomoura AT openoffice DOT org
+ COPYRIGHT   LGPLv3
+ 
+ 
+ FOL  ßàáâãäåæçèéêëìíîïðñòóôõöøùúûüýþÿ
+ LOW  ßàáâãäåæçèéêëìíîïðñòóôõöøùúûüýþÿ
+ UPP  ßÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØÙÚÛÜÝÞÿ
+ 
+ 
+ MIDWORD	'-.
+ 
+ 
  MAP 6
***************
*** 23,93 ****
  
! BREAK 68
! BREAK BREAK
! BREAK .
! BREAK ^-
! BREAK -$
! BREAK ^ex-
! BREAK ^Ex-
! BREAK ^EX-
! BREAK ^recém-
! BREAK ^Recém-
! BREAK ^RECÉM-
! BREAK ^pós-
! BREAK ^Pós-
! BREAK ^PÓS-
! BREAK ^pró-
! BREAK ^Pró-
! BREAK ^PRÓ-
! BREAK -alvo$
! BREAK -chave$
! BREAK -piloto$
! BREAK -padrão$
! BREAK -chefe$
! BREAK ^Vice-
! BREAK ^vice-
! BREAK ^VICE-
! BREAK ^além-
! BREAK ^aquém-
! BREAK ^Além-
! BREAK ^Aquém-
! BREAK ^não-
! BREAK ^Não-
! BREAK ^NÃO-
! BREAK -fêmeas$
! BREAK -fêmea$
! BREAK -macho$
! BREAK -machos$
! BREAK ^quase-
! BREAK ^Quase-
! BREAK ^QUASE-
! BREAK -teste$
! BREAK -Teste$
! BREAK -TESTE$
! BREAK -AC$
! BREAK -AL$
! BREAK -AP$
! BREAK -AM$
! BREAK -BA$
! BREAK -CE$
! BREAK -DF$
! BREAK -GO$
! BREAK -ES$
! BREAK -MA$
! BREAK -MT$
! BREAK -MS$
! BREAK -MG$
! BREAK -PA$
! BREAK -PB$
! BREAK -PR$
! BREAK -PE$
! BREAK -PI$
! BREAK -RJ$
! BREAK -RN$
! BREAK -RS$
! BREAK -RO$
! BREAK -RR$
! BREAK -SP$
! BREAK -SC$
! BREAK -SE$
! BREAK -TO$
  
--- 39,109 ----
  
! #BREAK 68
! #BREAK BREAK
! #BREAK .
! #BREAK ^-
! #BREAK -$
! #BREAK ^ex-
! #BREAK ^Ex-
! #BREAK ^EX-
! #BREAK ^recém-
! #BREAK ^Recém-
! #BREAK ^RECÉM-
! #BREAK ^pós-
! #BREAK ^Pós-
! #BREAK ^PÓS-
! #BREAK ^pró-
! #BREAK ^Pró-
! #BREAK ^PRÓ-
! #BREAK -alvo$
! #BREAK -chave$
! #BREAK -piloto$
! #BREAK -padrão$
! #BREAK -chefe$
! #BREAK ^Vice-
! #BREAK ^vice-
! #BREAK ^VICE-
! #BREAK ^além-
! #BREAK ^aquém-
! #BREAK ^Além-
! #BREAK ^Aquém-
! #BREAK ^não-
! #BREAK ^Não-
! #BREAK ^NÃO-
! #BREAK -fêmeas$
! #BREAK -fêmea$
! #BREAK -macho$
! #BREAK -machos$
! #BREAK ^quase-
! #BREAK ^Quase-
! #BREAK ^QUASE-
! #BREAK -teste$
! #BREAK -Teste$
! #BREAK -TESTE$
! #BREAK -AC$
! #BREAK -AL$
! #BREAK -AP$
! #BREAK -AM$
! #BREAK -BA$
! #BREAK -CE$
! #BREAK -DF$
! #BREAK -GO$
! #BREAK -ES$
! #BREAK -MA$
! #BREAK -MT$
! #BREAK -MS$
! #BREAK -MG$
! #BREAK -PA$
! #BREAK -PB$
! #BREAK -PR$
! #BREAK -PE$
! #BREAK -PI$
! #BREAK -RJ$
! #BREAK -RN$
! #BREAK -RS$
! #BREAK -RO$
! #BREAK -RR$
! #BREAK -SP$
! #BREAK -SC$
! #BREAK -SE$
! #BREAK -TO$
  
***************
*** 95,97 ****
  #   Número máximo de sugestões       
! MAXNGRAMSUGS 12
  
--- 111,113 ----
  #   Número máximo de sugestões       
! #MAXNGRAMSUGS 12
  
***************
*** 106,110 ****
  
! MAXDIFF 10
  
! ONLYMAXDIFF
  
--- 122,126 ----
  
! # MAXDIFF 10
  
! # ONLYMAXDIFF
  
