*** pt_PT.orig.aff	2015-12-06 19:21:14.846154146 -0200
--- pt_PT.aff	2015-12-06 19:21:14.850154146 -0200
***************
*** 1,6 ****
  SET UTF-8
! LANG pt_PT
! TRY aerisontcdmlupvgbfzáhçqjíxãóéêâúõACMPSBTELGRIFVDkHJONôywUKXZWQÁYÍÉàÓèÂÚ
! KEY qwertyuiop|asdfghjkl|zxcvbnm
! WORDCHARS -
  
--- 1,10 ----
  SET UTF-8
! #LANG pt_PT
! #TRY aerisontcdmlupvgbfzáhçqjíxãóéêâúõACMPSBTELGRIFVDkHJONôywUKXZWQÁYÍÉàÓèÂÚ
! #KEY qwertyuiop|asdfghjkl|zxcvbnm
! #WORDCHARS -
! 
! FOL  ßàáâãäåæçèéêëìíîïðñòóôõöøùúûüýþÿ
! LOW  ßàáâãäåæçèéêëìíîïðñòóôõöøùúûüýþÿ
! UPP  ßÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØÙÚÛÜÝÞÿ
  
***************
*** 1421,1433 ****
  
! MAP 11
! MAP aá
! MAP aã
! MAP aâ
! MAP eé
! MAP eê
! MAP ií
  MAP cç
! MAP oó
! MAP oô
! MAP oõ
! MAP uú
--- 1425,1432 ----
  
! MAP 6
! MAP aáãâAÁÃÂ
! MAP eéêEÉÊ
! MAP iíIÍ
  MAP cç
! MAP oóõôOÓÕÔ
! MAP uúüUÚÜ
